sdk/core/azure-core-amqp/inc/azure/core/amqp/internal/models/amqp_error.hpp

```diff
@@ -11,17 +11,17 @@
 #include <azure/core/internal/extendable_enumeration.hpp>
 
 struct ERROR_INSTANCE_TAG;
```

Comment:
QQ: How do you get rid of the ERROR_INSTANCE_TAG in the global namespace?

Part of the problem with the current set of changes is the forward declarations of uAMQP types in the global namespace.

Reply:
That's a separate problem, which probably exists even without this change?
But I see two options:
1. `#ifdef _azure_APIVIEW` ..., and figure out if we can hide that from ApiView.
2. use `void*` like Identity's `UniquePrivateKeyHelper` is using it:
https://github.com/Azure/azure-sdk-for-cpp/blob/f2bd22794d1990c48d066b85c930d33b99fe3465/sdk/identity/azure-identity/inc/azure/identity/client_certificate_credential.hpp#L27-L36
and then
https://github.com/Azure/azure-sdk-for-cpp/blob/f2bd22794d1990c48d066b85c930d33b99fe3465/sdk/identity/azure-identity/src/client_certificate_credential.cpp#L366-L373

Reply:
I can think of third possible option - ApiView to ignore forward declarations.